apps/backend/src/pies/domain/pie.ts

```diff
@@ -0,0 +1,28 @@
+import BigNumber from 'bignumber.js';
```

Comment:
This is a domain object representing a `Pie` which is decoupled from the database representation.